data_steward/bq_utils.py

```diff
@@ -91,12 +95,12 @@ def get_combined_deid_dataset_id():
 
     :return:  A name for the combined unidentified dataset id.
     """
-    dataset_id = os.environ.get(bq_consts.COMBINED_UNIDENTIFIED_DATASET,
+    dataset_id = os.environ.get(bq_consts.COMBINED_DEID_DATASET,
```

Comment:
Please remove the dataset name inferencing here so this is consistent with the other getters